modin/pandas/dataframe.py

```diff
@@ -1685,32 +1624,60 @@ def squeeze(self, axis=None):
         else:
             return self.copy()
 
-    def std(
-        self, axis=None, skipna=None, level=None, ddof=1, numeric_only=None, **kwargs
+    def _stat_operation(
+        self,
+        op_name: str,
+        axis: int,
```

Comment:
this also can be a string("index" or "columns")

Reply:
docstring and type hints fixed